Prusa-CW-Firmware/main.cpp

```diff
@@ -461,60 +492,58 @@ void setup() {
-  EEPROM.put(address++, washing_run_time);
-  EEPROM.put(address++, curing_run_time);
-  EEPROM.put(address++, finish_beep_mode);
-  EEPROM.put(address++, drying_run_time);
-  EEPROM.put(address++, sound_response);
-  EEPROM.put(address++, curing_machine_mode);
-  EEPROM.put(address++, heat_to_target_temp);
-  EEPROM.put(address++, target_temp_celsius);
-  EEPROM.put(address++, target_temp_fahrenheit);
-  EEPROM.put(address++, SI_unit_system);
-  EEPROM.put(address++, heater_failure);
-  EEPROM.put(address++, FAN1_CURING_SPEED);
-  EEPROM.put(address++, FAN1_DRYING_SPEED);
-  EEPROM.put(address++, FAN1_PREHEAT_SPEED);
-  EEPROM.put(address++, FAN2_CURING_SPEED);
-  EEPROM.put(address++, FAN2_DRYING_SPEED);
-  EEPROM.put(address++, FAN2_PREHEAT_SPEED);
-  EEPROM.put(address++, resin_preheat_run_time);
-  EEPROM.put(address++, resin_target_temp_celsius);
+void write_config(unsigned int address) {		//useless address param
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->magic)), magic2);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->washing_speed)), washing_speed);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->curing_speed)), curing_speed);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->washing_run_time)), washing_run_time);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->curing_run_time)), curing_run_time);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->finish_beep_mode)), finish_beep_mode);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->drying_run_time)), drying_run_time);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->sound_response)), sound_response);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->curing_machine_mode)), curing_machine_mode);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->heat_to_target_temp)), heat_to_target_temp);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->target_temp_celsius)), target_temp_celsius);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->target_temp_fahrenheit)), target_temp_fahrenheit);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->SI_unit_system)), SI_unit_system);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->heater_failure)), heater_failure);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->FAN1_CURING_SPEED)), FAN1_CURING_SPEED);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->FAN1_DRYING_SPEED)), FAN1_DRYING_SPEED);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->FAN1_PREHEAT_SPEED)), FAN1_PREHEAT_SPEED);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->FAN2_CURING_SPEED)), FAN2_CURING_SPEED);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->FAN2_DRYING_SPEED)), FAN2_DRYING_SPEED);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->FAN2_PREHEAT_SPEED)), FAN2_PREHEAT_SPEED);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->resin_preheat_run_time)), resin_preheat_run_time);
+  EEPROM.put(reinterpret_cast<int>(&(eeprom_base->resin_target_temp_celsius)), resin_target_temp_celsius);
 }
 
-
 void read_config(unsigned int address) {
```

Comment:
main.cpp:520:31: warning: unused parameter 'address' [-Wunused-parameter]
 void read_config(unsigned int address) {

Please fix this compiler warning. (Remove unused function parameter)